### PR TITLE
chore(deps): update dev deps + ci

### PR DESCRIPTION
**Pull request by stephenlautier**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,26 +44,26 @@
   },
   "devDependencies": {
```
Comment (claylaut) [inline package.json:45]: bump version

**Review by claylaut — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (stephenlautier) [reply package.json:45]: done

**Review by claylaut — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -29,20 +29,23 @@ jobs:
       - run:
```
Comment (claylaut) [inline .circleci/config.yml:29]: update to `image: circleci/node:12.4.0`

**Review by claylaut — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (stephenlautier) [reply .circleci/config.yml:29]: done

**Review by claylaut — APPROVED** [review]

**Pull request merged**